example/src/main/java/org/apache/rocketmq/example/simple/LitePullConsumerAsync.java

```diff
@@ -0,0 +1,42 @@
+package org.apache.rocketmq.example.simple;
```

Comment:
IMO, no need for a new example.

Reply:
Yes, I totally agree with you so I'll remove this example tonight. And I want to make sure my understanding is correct. 

Since we haven't figure out a suitable way to start an Async pull, we can only change the constructor method of PullTaskImpl in seek(). Therefore, in users' view (example view), they just call the seek() function. Maybe we can add a new example after we separate the Async pull and Sync pull.